kafka-connect-s3/src/test/java/io/confluent/connect/s3/TopicPartitionWriterTest.java

```diff
@@ -491,6 +495,53 @@ public void testWriteRecordTimeBasedPartitionWithNullTimestamp() throws Exceptio
     topicPartitionWriter.write();
   }
 
+  /* Test will start writing before the hour, and write some records after the hour, and verify
+     there is still only one output file created.
```

Comment:
Nit: block comments should have a `*` at the beginning of each line, similar to how JavaDoc comments do.```suggestion
   * there is still only one output file created.
```